js/WeaterApp.js

```diff
@@ -67,108 +66,123 @@ class WeatherApp {
-
-		const sunrise = this.weatherData.sys.sunrise;
-		const sunset = this.weatherData.sys.sunset;
-		const time = parseInt(moment().format('X'));
-		const id = this.weatherData.weather[0].id;
-
-		this.isDay = (time > sunrise && time < sunset) ? true : false;
-		const iconSet = (this.isDay) ? icons.dayIcons : icons.nightIcons;
-		const iconBG = (!this.isDay) ? 'night' : '';
-
-
+		this.getGeneralInfo();
+		this.getTempertures();
+		this.getWind();
+		this.getIconSet();
+		this.getBackground();
 
 		this.infoApp.innerHTML = `
 			<div>
-				<span class="icon-location1"></span> You are in: ${city}, ${country}
+				<span class="icon-location1"></span><span>You are in: ${this.city}, ${this.country}</span>
 			</div>
 			<div class="wrapper">
-				<p class="date">Today is ${currentDate}</p>
+				<p class="date">Today is ${this.currentDate}</p>
+
 				<div class="info-wrapper">
-					<p class="small ${iconBG}">
+					<p class="small ${this.iconBG}">
 						<img width="35" src="./svg/thermometer.svg" alt="celsius degree" />
-						<span>Day ${tempMaxC}º <span class="icon icon-arrow-up"></span></span>
-						<span>Night ${tempMinC}º <span class="icon icon-arrow-down"></span></span>
+						<span>Day ${this.tempMaxC}º <span class="icon icon-arrow-up"></span></span>
+						<span>Night ${this.tempMinC}º <span class="icon icon-arrow-down"></span></span>
 					</p>
 				</div>
+
 				<div class="current">
 					<div id="icon">
-						<p class="weather-icon"><img width="100" src="./svg/${iconSet[id]}"></img></p>
-						<span class="small weather-desc">${description}</span>
+						<p class="weather-icon"><img width="100" src="./svg/${this.iconSet[this.id]}"></img></p>
+						<span class="small weather-desc">${this.description}</span>
 					</div>
 					<div>
-						<p class="big">${temp}º<span class="superindex">c</span></p>
-						<span>Feels Like ${feels}º</span>
+						<p class="big">${this.temp}º<span class="superindex">c</span></p>
+						<span>Feels Like ${this.feels}º</span>
 					</div>
 				</div>
+
 				<div class="wrapper-info">
-					<p class="small ${iconBG}">
-						<img width="35" src="${windIcon}" alt="wind-icon"/>
-						Wind: ${windKMH} km/h from ${windDir}
+					<p class="small ${this.iconBG}">
+						<img width="35" src="${this.windIcon}" alt="wind-icon"/>
+						<span>Wind: ${this.windKMH} km/h from ${this.windDir}</span>
 					</p>
-					<p class="small ${iconBG}">
+					<p class="small ${this.iconBG}">
 						<img width="35" src="./svg/humidity.svg" alt="humidity-icon"/>
-						Humidity: ${humidity} %</p>
-					<p class="small ${iconBG}">
+						<span>Humidity: ${this.humidity} % </span>
+					</p>
+					<p class="small ${this.iconBG}">
 						<img width="35" src="./svg/barometer.svg" alt="barometer-icon"/>
-						Pressure: ${pressure} hPa
+						<span>Pressure: ${this.pressure} hPa</span>
 					</p>
 				</div>
-
 			</div>
 		`;
 
 		this.input.setAttribute("placeholder", " try another city");
```

Comment:
Tenés un espacio de más acá.

```suggestion
		this.input.setAttribute("placeholder", "try another city");
```